api/app/signals/apps/signals/models/category_assignment.py

```diff
@@ -43,4 +49,5 @@ def save(self, *args, **kwargs):
         # Note: this may be moved to the API layer of SIA/Signalen.
         self.deadline, self.deadline_factor_3 = DeadlineCalculationService.from_signal_and_category(
             self._signal, self.category)
+        self.stored_handling_message = self.category.handling_message  # SIG-3555
```

Comment:
I think this is something that we need to do in the SignalManager. Now it is not possible to set the handling message when the category assignment updates.

Reply:
As discussed, we can leave this in the `save` for now and later do a refactor that cleans up all model `saves`.